src/Features/Core/Portable/CodeRefactorings/IRefactoringHelpersService.cs

```diff
@@ -34,5 +34,6 @@ internal interface IRefactoringHelpersService : ILanguageService
         /// </para>
         /// </summary>
         Task<TSyntaxNode> TryGetSelectedNodeAsync<TSyntaxNode>(Document document, TextSpan selection, CancellationToken cancellationToken) where TSyntaxNode : SyntaxNode;
+        Task<TSyntaxNode> TryGetSelectedNodeAsync<TSyntaxNode>(CodeRefactoringContext context) where TSyntaxNode : SyntaxNode;
```

Comment:
can be extension.

Reply:
I'll reconcile that once your extension method & everything gets properly merged in.